dockerfiles/platform-core-debug/Dockerfile

```diff
@@ -28,7 +28,7 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en  
 ENV LC_ALL en_US.UTF-8     
 
-RUN apt-get install -y libcap2-bin zip libarchive13
+RUN apt-get install -y libcap2-bin zip libarchive13 libatlas-base-dev
```

Comment:
Not sure why the debug version would need a different package to the release one

Reply:
Thought adding "-dev" will help debug. Otherwise I can just add the same one as release version

Reply:
I change platform-core-debug to have the same libatlas3-base as platform-core. If later someone requests dev package we can make a change